### PR TITLE
Odyssey tab, tag, color, monospace font and ui-indicator updates

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
- [x] Tabs: now uses 16px base font instead of 14px. This now matches the design.
- [x] Tabs: unused CSS custom properties that were used for moving tab indicator implementation have been removed.
- [x] Tabs: Border now lives on the tablist, rather than the tabpanel.
- [x] Tabs: tabpanel padding updated to match the design.
- [x] Tag: tag--list now `inline-block` by default
- [x] Tag: Background color updated to gray 100, per design
- [x] Toast/Modal: Close UI indicator updated to use variant from odyssey-icon
- [x] Now uses Incosolata as the default monospace font for Odyssey, this was recently decided by the brand team! 
- [x] Adds turquoise 400

### CODE REVIEW

File: packages/odyssey/src/scss/abstracts/_functions.scss
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/abstracts/_functions.scss:75]: Does this still need to go through SVGO? (the `width/height` should have been stripped if so)

File: packages/odyssey/src/scss/abstracts/_tokens.scss
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/abstracts/_tokens.scss:17]: Good with this even if we can't ship it at present - progressive enhancement where available.

File: packages/odyssey/src/scss/components/_tag.scss
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/components/_tag.scss:25]: I don't think we should change this to account for a horizontal label unless we're going to ship the label itself or include additional guidance around Parents. I'd prefer if we added a Horiz variant / horiz support separately.

File: packages/odyssey/src/scss/abstracts/_functions.scss
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss:75]: Nope, It should have been updated - I ran it through the UI. Will give it another look.

File: packages/odyssey/src/scss/components/_tag.scss
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/components/_tag.scss:25]: My understanding is that this is as @joejezowski-okta had intended for it to be by default, per the design. I will revert this for now and will schedule a follow-up chat/log an issue.

File: packages/odyssey/src/scss/abstracts/_functions.scss
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss]: Fixed.

**Review by edburyenegren-okta — APPROVED** [review]
🚀 

thanks for being flexible while we nail down a lot of moving parts @arnoldsandoval-okta

File: packages/odyssey/src/scss/abstracts/_functions.scss
Comment (santhoshbalakrishnan) [inline packages/odyssey/src/scss/abstracts/_functions.scss:78]: Curious, why are viewbox sizes different for different icons? Do we want to standardize on similar viewboxes and modify the content within in relatively?

**Review by santhoshbalakrishnan — APPROVED** [review]
🍰

File: packages/odyssey/src/scss/abstracts/_functions.scss
Comment (edburyenegren-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss:78]: Currently, they're all a little bespoke - @joejezowski-okta designed them for very specific use cases. Agreed that we should work to create some standards here, though. Worth investigating after we revisit Type, @joejezowski-okta @arnoldsandoval-okta?

Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/abstracts/_functions.scss:78]: Im good with it if Joe is. @joejezowski-okta let me know and I can log an issue for this!

**Comment by arnoldsandoval-okta** [comment]
We were holding off on merging this until #731 was approved. Spoke to @jmelberg-okta and he agreed it was fine to merge this in despite that PR not having final approval as he has signed off for the weekend.

**Pull request merged**
